client/wfprev-war/src/main/angular/src/app/services/token.service.ts

```diff
@@ -222,6 +222,18 @@ export class TokenService {
     return fullName || null;
   }
 
+  public getUserFullNameLastFirst(): string | null {
```

Comment:
The function below has a load of failure cases (name order,, multi space, multi word names, etc). Just use the names as provided from the token??

```
public getUserFullNameLastFirst(): string | null {
    const first = (this.tokenDetails?.given_name ?? this.tokenDetails?.givenName) ?? '';
    const last = (this.tokenDetails?.family_name ?? this.tokenDetails?.familyName) ?? '';
    const fullName = `${last}, ${first}`.trim();
    return fullName || null;
}
```